### PR TITLE
Added a test suite for Pacific Tier 2 scale testing

### PR DESCRIPTION
**Pull request by manasagowri**
# Description

Success execution logs - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-H5LERH/

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs

# Checklist:

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarin Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test

### CODE REVIEW

File: suites/pacific/cephadm/tier1_scale_testing.yaml [hunk 1]
```diff
@@ -0,0 +1,511 @@
+tests:
```
[the remark this answers is not in the document]
Comment (manasagowri) [reply suites/pacific/cephadm/tier1_scale_testing.yaml]: Done. Name changed to tier_2_cluster_scaleup_scaledown.yaml

File: suites/pacific/cephadm/tier1_scale_testing.yaml [hunk 2]
```diff
@@ -0,0 +1,511 @@
...
+  - test:
+      name: Install ceph pre-requisites
+      desc: installation of ceph pre-requisites
+      module: install_prereq.py
+      abort-on-fail: true
+  - test:
+      name: Cluster deployment
```
[the remark this answers is not in the document]
Comment (manasagowri) [reply suites/pacific/cephadm/tier1_scale_testing.yaml]: Done.

File: suites/pacific/cephadm/tier1_scale_testing.yaml [hunk 3]
```diff
@@ -0,0 +1,511 @@
...
+              command: apply
+              service: crash
+              base_cmd_args:
+                verbose: true
+              args:
+                placement:
+                  nodes: "*"
+      destroy-cluster: false
+      abort-on-fail: true
+  - test:
```
[the remark this answers is not in the document]
Comment (manasagowri) [reply suites/pacific/cephadm/tier1_scale_testing.yaml]: Done.

File: suites/pacific/cephadm/tier_2_cluster_scaleup_scaledown.yaml [hunk 4]
```diff
@@ -0,0 +1,624 @@
...
+      desc: Perform export during read/write,resizing,flattening,lock operations
+
+  - test:
+      name: rgw sanity tests
+      module: sanity_rgw.py
+      config:
+        script-name: test_multitenant_user_access.py
+        config-file-name: test_multitenant_access.yaml
+        timeout: 300
+      desc: Perform rgw tests
```
Comment (sunilkumarn417) [inline suites/pacific/cephadm/tier_2_cluster_scaleup_scaledown.yaml:624]: please end with newline.

File: suites/pacific/cephadm/tier_2_cluster_scaleup_scaledown.yaml [hunk 5]
```diff
@@ -0,0 +1,624 @@
+tests:
```
Comment (sunilkumarn417) [inline suites/pacific/cephadm/tier_2_cluster_scaleup_scaledown.yaml:1]: please add the test case description and details in the comment part.

**Review by sunilkumarn417 — COMMENTED** [review]
Hi @manasagowri 
can you please address on few minor comments.

File: suites/pacific/cephadm/tier_2_cluster_scaleup_scaledown.yaml (hunk 4 above)
Comment (manasagowri) [reply suites/pacific/cephadm/tier_2_cluster_scaleup_scaledown.yaml:624]: Done

File: suites/pacific/cephadm/tier_2_cluster_scaleup_scaledown.yaml (hunk 5 above)
Comment (manasagowri) [reply suites/pacific/cephadm/tier_2_cluster_scaleup_scaledown.yaml:59]: Done.

**Review by psathyan — APPROVED** [review]
Looks good to me

**Review by sunilkumarn417 — APPROVED** [review]

**Pull request merged**